python/paddle/fluid/contrib/slim/quantization/qat2_int8_mkldnn_pass.py

```diff
@@ -83,16 +83,14 @@ def apply(self, graph):
         graph = self._propagate_scales(graph)
         graph = self._set_dummy_out_scales(graph)
         graph = self._quantize_fp32_graph(graph)
+        graph = self._optimize_int8_graph(graph)
         graph = self._cleanup(graph)
         return graph
 
     def apply_fp32(self, graph):
         assert isinstance(graph,
                           IrGraph), 'graph must be the instance of IrGraph.'
 
-        graph = self._gather_scales(graph)
-        graph = self._remove_fake_ops(graph)
-        graph = self._dequantize_weights(graph)
```

Comment:
We no longer need to dequantize_weights? When converting the model to pure FP32?

Reply:
No, now we assume the FP32 model is a plain FP32 one and does not need dequantization.